rtv/content.py

```diff
@@ -122,7 +122,7 @@ def strip_praw_submission(sub):
         data['score'] = '{} pts'.format(sub.score)
         data['author'] = name
         data['permalink'] = sub.permalink
-        data['subreddit'] = strip_subreddit_url(sub.permalink)
+        data['subreddit'] = str(sub.subreddit)
```

Comment:
OK, I've had a look into this and it's ok, the `Subreddit` object you get from `Submission.subreddit` is not "fetched" and has been initialised with the subreddit name and that is all that is returned from `Subreddit.__str__()`.

A lot of the calls on the `Subreddit` object *will* cause the cache to be refreshed (or initialised) but the subreddit name is not one of them. `__str__()` also does not cause the cache to be updated as it's just returning the name of the Sub, which is a param to `__init__()` passed to it by the `Submission` object.

In short, this is fine, but I understand your hesitance.